utils/policy_roles_adapter.ts

```diff
@@ -1,40 +1,46 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {Role} from '@mattermost/types/roles';
 import {Permissions} from 'mattermost-redux/constants/index';
 
 const MAPPING = {
     enableTeamCreation: {
-        true: [{roleName: 'system_user', permission: Permissions.CREATE_TEAM, shouldHave: true}],
-        false: [{roleName: 'system_user', permission: Permissions.CREATE_TEAM, shouldHave: false}],
+        True: [{roleName: 'system_user', permission: Permissions.CREATE_TEAM, shouldHave: true}],
```

Comment:
why did we change from 'true' to 'True'

Reply:
Here true is creating conflicts, as it is a string as  well as Boolean.
So it's creating type mismatch errors in some places

Reply:
Here true is creating conflicts, as it is a string as  well as Boolean.
So it's creating type mismatch errors in some places

Reply:
ahh but its changing the behavior.

Reply:
another option was to add single quotes around true and disable lint warnings for the line, and give the type **'true' | 'false'** wherever required.

Reply:
@AbhinavVihan you can do ['true']